src/main/java/sg/ncl/MainController.java

```diff
@@ -328,7 +328,8 @@ public String redirectNotFound(HttpSession session) {
     }
 
     @RequestMapping(value = "/login", method = RequestMethod.GET)
-    public String login(Model model) {
+    public String login(Model model, HttpSession session) {
+//        removeSessionVariables(session);
         model.addAttribute("loginForm", new LoginForm());
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3ACommentedOutCodeLine)